packages/contracts/scripts/deploy.ts

```diff
@@ -38,11 +40,12 @@ async function deploy(key: string) {
       `export declare const ${network.name.toUpperCase()}_${key}_ADDRESS: string;\n`
     );
   }
+  return contract.address;
 }
 
 async function main() {
   await deploy("PASSPORT_FACTORY");
-  await deploy("STAKING");
+  await deploy("TEST_TOKEN", "10000").then((addr) => deploy("STAKING", addr));
```

Comment:
when we eventually launch on main, we'll need to change this to point to the cabin token